src/prerequisites/tests/test_signals.py

```diff
@@ -154,3 +156,42 @@ def test_update_cache_triggered_by_parent_object_deletion(self, task):
 
         quest.delete()  # doesn't call task because parent_object no longer exists.
         self.assertEqual(task.call_count, 1)
+
+    def test_on_quest_badge_save_with_rank_prereq__creation(self):
```

Comment:
Test looks good, but there's no test to confirm:
- invert=True doesn't make the map
- pre-existing map is handled gracefully

Reply:
done